crates/topos/src/main.rs

```diff
@@ -134,6 +129,40 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     }
 }
 
+fn build_resources() -> Vec<KeyValue> {
+    let mut resources = Vec::new();
+
+    resources.push(KeyValue::new(
+        "service.name",
+        std::env::var("TCE_JAEGER_SERVICE_NAME").expect("TCE_JAEGER_SERVICE_NAME must be defined"),
```

Comment:
`TCE_JAEGER_SERVICE_NAME` is optional, you should not use expect here

Reply:
Actually not to complicate things it will be handled in another PR #184

Reply:
Yes I just reused the existing code